rke/k8s_version_info.go

```diff
@@ -643,6 +643,10 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
 			MinRancherVersion: "2.6.7-patch0",
 			MinRKEVersion:     "1.3.13-rc0",
 		},
+		"v1.25": {
+			MinRancherVersion: "2.7.1-patch0",
+			MinRKEVersion:     "1.4.0-rc0",
```

Comment:
There is no rke 1.4.0-rc0. Why are we using that? The latest version of rke is 1.4.0-rc4 https://github.com/rancher/rke/tags. Is the general policy to use an rc here for the min version or what docs say what to use for 1.25?

Reply:
It's a general pattern we follow, we need to use RCs so the version loads for testing. 
@jiaqiluo let's make sure this doesn't land into the RKE version which will get tagged with out of band KDM release for November.

Reply:
> @jiaqiluo let's make sure this doesn't land into the RKE version which will get tagged with out of band KDM release for November.

No problem, I will hold this PR.